code/modules/clothing/spacesuits/miscellaneous.dm

```diff
@@ -68,7 +68,7 @@
 	flags_inv = 0
 	w_class = 3
 	allowed = list(/obj/item/weapon/gun,/obj/item/ammo_box,/obj/item/ammo_casing,/obj/item/weapon/melee/baton,/obj/item/weapon/restraints/handcuffs,/obj/item/weapon/tank/internals)
-	armor = list(melee = 80, bullet = 80, laser = 50,energy = 50, bomb = 100, bio = 100, rad = 100)
+	armor = list(melee = 50, bullet = 40, laser = 30,energy = 50, bomb = 50, bio = 100, rad = 100)

```

Comment:
I guess deathsquads and whether they should be mortal or not is a completely different argument from normal armor values. I still think invincible mobs shouldn't be a regular thing.